xchange-bithumb/src/main/java/org/knowm/xchange/bithumb/dto/BithumbResponse.java

```diff
@@ -0,0 +1,36 @@
+package org.knowm.xchange.bithumb.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.StringUtils;
+
+public class BithumbResponse<D> {
+
+  private final String status;
+  private final String message;
+  private final D data;
+
+  public BithumbResponse(
+      @JsonProperty("status") String status,
+      @JsonProperty("message") String message,
+      @JsonProperty("data") D data) {
```

Comment:
if you'd add this here:

```
if (!StringUtils.equals(status, "0000")) {
    throw new ExceptionalReturnContentException("Got error status");
}
```

then  rescu will catch it and re-parse the body as BithumbException

Reply:
@walec51 Thanks, I fixed this, too.